test/utility/misc.c

```diff
@@ -323,13 +325,90 @@ testutil_create_backup_directory(const char *home)
     size_t len;
     char *cmd;
 
-    len = strlen(home) * 4 + strlen(HOME_BACKUP_INIT_CMD) + 1;
+    len = strlen(home) * 6 + strlen(HOME_BACKUP_INIT_CMD) + 1;
     cmd = dmalloc(len);
-    testutil_check(__wt_snprintf(cmd, len, HOME_BACKUP_INIT_CMD, home, home, home, home));
+    testutil_check(
+      __wt_snprintf(cmd, len, HOME_BACKUP_INIT_CMD, home, home, home, home, home, home));
     testutil_checkfmt(system(cmd), "%s", "backup directory creation failed");
     free(cmd);
 }
 
+/*
+ * testutil_verify_src_backup --
+ *     Verify a backup source home directory against a backup directory for changes to blocks that
```

Comment:
Is it worth mentioning that this checks both backup IDs, but assumes that the "backup" directory corresponds to only one of them?

Reply:
I'll modify the content here after adding in the ID arg.